game/src/sandbox/speed.rs

```diff
@@ -390,23 +402,16 @@ impl JumpToTime {
                     0.25 * ctx.canvas.window_width,
                     target.to_percent(end_of_day).min(1.0),
                 )
-                .named("time slider"),
-                Btn::text_bg2(format!("Jump to {}", target.ampm_tostring()))
-                    .build(ctx, "jump to time", hotkey(Key::Enter))
-                    .centered_horiz()
-                    .named("jump to time"),
-                Widget::horiz_separator(ctx, 0.25).margin_above(10),
-                Btn::text_bg2("Jump to the next delay over 5 minutes")
-                    .build_def(ctx, None)
-                    .centered_horiz(),
-                Checkbox::checkbox(
-                    ctx,
-                    "don't draw (for faster simulations)",
-                    None,
-                    app.opts.dont_draw_time_warp,
-                )
-                .margin_above(30)
-                .named("don't draw"),
+                .named("time slider")
+                .centered_horiz()
+                // EZGUI FIXME: margin_below having no effect here, so instead we add a margin_top
```

Comment:
This is because `Widget::col` overwrites `margin_below` for all its members. Maybe it ought to only overwrite if the margin is unset? That could maybe whittle down some callers of `custom_col`